packages/dao/contracts/Api3Template.sol

```diff
@@ -32,25 +34,28 @@ contract Api3Template is Api3BaseTemplate {
     * @dev Deploy an authoritative DAO using the API3 Staking Pool
     * @param _id String with the name for org, will assign `[id].aragonid.eth`
     * @param _api3Pool Address of the API3 staking pool, supplies voting power
-    * @param _votingSettings Array of [supportRequired, minAcceptanceQuorum, voteDuration] to set up the voting app of the organization NOTE: deleted minProposerPower
+    * @param _mainVotingSettings Array of [supportRequired, minAcceptanceQuorum, voteDuration] to set up the voting app of the organization NOTE: deleted minProposerPower
+    * @param _secondaryVotingSettings Array of [supportRequired, minAcceptanceQuorum, voteDuration] to set up the voting app of the organization NOTE: deleted minProposerPower
     * @param _permissionManager The administrator that's initially granted control over the DAO's permissions
     */
     function newInstance(
         string memory _id,
         MiniMeToken _api3Pool,
-        uint64[3] memory _votingSettings,
+        uint64[3] memory _mainVotingSettings,
+        uint64[3] memory _secondaryVotingSettings,
         address _permissionManager
     )
     public
     {
         require(_api3Pool != address(0), "Invalid API3 Voting Rights");
 
         _validateId(_id);
-        _validateVotingSettings(_votingSettings);
+        _validateVotingSettings(_mainVotingSettings);
+        _validateVotingSettings(_secondaryVotingSettings);
 
         (Kernel dao, ACL acl) = _createDAO();
-        (Api3Voting voting, Agent agent) = _setupApps(
-            dao, acl, _api3Pool, _votingSettings, _permissionManager
+        (Api3Voting mainVoting, Api3Voting secondaryVoting, Agent mainAgent, Agent secondaryAgent) = _setupApps(
+            dao, acl, _api3Pool, _mainVotingSettings,_secondaryVotingSettings, _permissionManager
         );
         _transferRootPermissionsFromTemplateAndFinalizeDAO(dao, _permissionManager);
```

Comment:
What is this `permissionManager` thing ? Burak asked you tu pass a special address as the manager of all the permissions ?

Reply:
That was here before we were involved into the project, and to be honest I am not sure if we can change this...

Reply:
I think it would be worth to ask Burak because that's an important security issue. If this `permissionManager` is unsafe the whole DAO is.

Reply:
No idea what a `permissionManager` is and what it does.

Reply:
Let's use the most secure voting app address instead of this address then - everywhere it is used.